charts/flyte-core/templates/_helpers.tpl

```diff
@@ -102,28 +102,6 @@ app.kubernetes.io/managed-by: {{ .Release.Service }}
 {{- end }}
 {{- end -}}
 
-{{- define "flyteagent.name" -}}
-flyteagent
-{{- end -}}
-
-{{- define "flyteagent.selectorLabels" -}}
-app.kubernetes.io/name: {{ template "flyteagent.name" . }}
-app.kubernetes.io/instance: {{ .Release.Name }}
-{{- end -}}
-
-{{- define "flyteagent.labels" -}}
-{{ include "flyteagent.selectorLabels" . }}
-helm.sh/chart: {{ include "flyte.chart" . }}
-app.kubernetes.io/managed-by: {{ .Release.Service }}
-{{- end -}}
-
-{{- define "flyteagent.podLabels" -}}
-{{ include "flyteagent.labels" . }}
-{{- with .Values.flyteagent.podLabels }}
-{{ toYaml . }}
-{{- end }}
-{{- end -}}
-
```

Comment:
Is this intentional?

Reply:
yes since we move the agent chart to `flyte-agent/`